recordset/recordset.html

```diff
@@ -2,101 +2,101 @@
-            <!-- pagination -->
-            <div style="margin-bottom: 10px;">
-                <span>{{ pageInfo.recordStart }} to {{ pageInfo.recordEnd }} of {{ vm.count }} Items</span>
-                <span class="dropdown" style="margin-left: 5px">
-                    <button class="btn btn-sm btn-default dropdown-toggle" type="button" data-toggle="dropdown">Items per page
-                        <span class="caret"></span></button>
-                    <ul class="dropdown-menu">
-                        <li><a href ng-click="pageLimit(10)">
-                            <span ng-show="pageInfo.pageLimit === 10" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 10" class="glyphicon glyphicon-invisible"></span>
-                            10</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(25)">
-                            <span ng-show="pageInfo.pageLimit === 25" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 25" class="glyphicon glyphicon-invisible"></span>
-                            25</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(50)">
-                            <span ng-show="pageInfo.pageLimit === 50" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 50" class="glyphicon glyphicon-invisible"></span>
-                            50</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(75)">
-                            <span ng-show="pageInfo.pageLimit === 75" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 75" class="glyphicon glyphicon-invisible"></span>
-                            75</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(100)">
-                            <span ng-show="pageInfo.pageLimit === 100" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 100" class="glyphicon glyphicon-invisible"></span>
-                            100</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(200)">
-                            <span ng-show="pageInfo.pageLimit === 200" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 200" class="glyphicon glyphicon-invisible"></span>
-                            200</a>
-                        </li>
-                    </ul>
-                </span>
-            </div>
+        <!-- loading spinner -->
+        <div ng-show="pageInfo.loading" id="spinner">
+            <img src="spinner.gif" style="display: block; margin: 0 auto; width: 32px; height: 32px; opacity: 0.8;"/>
+            <div style="margin-top: 15px;">Loading...</div>
+        </div>
```

Comment:
@hongsudt Here it is, for you to assign as needed: https://github.com/informatics-isi-edu/chaise/issues/550.